### PR TITLE
Perception: Changed lane assignment logic: Used closest point for lat…

### PR DESCRIPTION
**Pull request by techoe**
Perception: Changed lane assignment logic: Used closest point for lateral distance; Removed CompensateLaneObjects

### CODE REVIEW

File: modules/perception/obstacle/camera/lane_post_process/cc_lane_post_processor/cc_lane_post_processor.cc [hunk 1]
```diff
@@ -806,62 +825,81 @@ bool CCLanePostProcessor::Process(const cv::Mat &lane_map,
...
               origin_lateral_dist_object_id.end(),
               CompOriginLateralDistObjectID);
-    int i_l0 = -1;
+    int index_closest_left = -1;
+    int index_closest_right = -1;
     for (int k = 0; k < count_lane_objects; ++k) {
       if (origin_lateral_dist_object_id[k].first >= 0) {
-        i_l0 = k;
+        index_closest_left = k;
+        if (k > 0) {
```
Comment (gchen-apollo) [inline modules/perception/obstacle/camera/lane_post_process/cc_lane_post_processor/cc_lane_post_processor.cc]: CI didn't pass 
and I think line 845 may not be correct.  maybe it should be:    if (k < count_lane_objects-1)   ?

File: modules/perception/obstacle/camera/lane_post_process/cc_lane_post_processor/cc_lane_post_processor.cc [hunk 2]
```diff
@@ -793,6 +810,8 @@ bool CCLanePostProcessor::Process(const cv::Mat &lane_map,
           }
         }
       }
+      // left most point - C0 < 2.5
+      // *** TO DO *** use C0 or closest_x point for the distance measure.
```
Comment (lianglia-apollo) [inline modules/perception/obstacle/camera/lane_post_process/cc_lane_post_processor/cc_lane_post_processor.cc]: please use TODO(name) format for tasks to be done in the future.

File: modules/perception/obstacle/camera/lane_post_process/cc_lane_post_processor/cc_lane_post_processor.cc (hunk 1 above)
Comment (techoe) [reply modules/perception/obstacle/camera/lane_post_process/cc_lane_post_processor/cc_lane_post_processor.cc]: THank you, This is incorrect statement.

File: modules/perception/obstacle/camera/lane_post_process/cc_lane_post_processor/cc_lane_post_processor.cc (hunk 2 above)
Comment (techoe) [reply modules/perception/obstacle/camera/lane_post_process/cc_lane_post_processor/cc_lane_post_processor.cc]: Thank you for the review and advice. I will follow the format!

**Pull request merged**